### ISSUE TITLE
Missing default config is not detected properly

### ISSUE THREAD

**Issue by jvyden** [LittleBigRefresh/FreshPresence#2]
```zig
[jvyden@the-overcooler bin]$ ./FreshPresence 
error: FileNotFound
/usr/lib/zig/lib/std/os.zig:1719:23: 0x2c647b in openatZ (FreshPresence)
            .NOENT => return error.FileNotFound,
                      ^
/usr/lib/zig/lib/std/fs.zig:1534:25: 0x2c5d92 in realpathZ (FreshPresence)
            else => |e| return e,
                        ^
/usr/lib/zig/lib/std/fs.zig:1520:9: 0x2c6bc4 in realpath (FreshPresence)
        return self.realpathZ(&pathname_c, out_buffer);
        ^
/usr/lib/zig/lib/std/fs.zig:1608:35: 0x2c7131 in realpathAlloc (FreshPresence)
        return allocator.dupe(u8, try self.realpath(pathname, buf[0..]));
                                  ^
/home/jvyden/Documents/FreshPresence/src/config.zig:29:21: 0x2caf25 in getConfig (FreshPresence)
    var full_path = try (config_dir orelse cwd).realpathAlloc(allocator, config_filename);
                    ^
/home/jvyden/Documents/FreshPresence/src/main.zig:31:20: 0x2d74c0 in runApp (FreshPresence)
    const config = try Config.getConfig(allocator);
                   ^
/home/jvyden/Documents/FreshPresence/src/main.zig:26:9: 0x2dce2c in main (FreshPresence)
        return err;
        ^
```

**Comment by Beyley** [comment]
oop, must have broken that in the latest commits, will check it out

**Comment by Beyley** [comment]
Should have been fixed by #3d4cb80
